modules/cactus-web/src/Toggle/Toggle.tsx

```diff
@@ -111,12 +110,14 @@ const Switch = styled.div`
   }
 
   input:checked ~ & {
-    background-color: ${(p): string => p.theme.colors.success};
-
     ::after {
       left: 26px;
     }
   }
+
+  input:checked:not([disabled]) ~ & {
```

Comment:
I think it'd be a bit more consistent to use the pseudo-class:
```suggestion
  input:checked:not(:disabled) ~ & {
```